book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
```

Comment:
- [ ] kindly make sure the rentals is accessible outside the Book Class, which means you should add getters and setters for the rentals array. Hint `attr_accessor :rentals`